pkgs/development/tools/misc/src-cli/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+let
+  owner = "sourcegraph";
+  repo = "src-cli";
+in
+buildGoModule rec {
+  pname = "src-cli";
+  version = "4.3.0";
+
+  src = fetchFromGitHub {
+    inherit owner repo;
+    rev = "${version}";
+    sha256 = "sha256-mhQdmfObeR7obd+o3emBf1GXIqVZAZCFNQGsG1a06+I=";
+  };
```

Kind of opinionated...but something like is more aligned with the current style.

```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
}:

buildGoModule rec {
  pname = "src-cli";
  version = "4.3.0";

  src = fetchFromGitHub {
    owner = "sourcegraph";
    repo = "src-cli";
    rev = "${version}";
    sha256 = "sha256-mhQdmfObeR7obd+o3emBf1GXIqVZAZCFNQGsG1a06+I=";
  };
```